### PR TITLE
Feature/build out dashboard

### PR DESCRIPTION
**Pull request by jon-schlandt**
## Is this a fix or a feature?
This is a feature. I built out a concept for the Dashboard component and its children (TeeTimeContainer and TeeTime). Currently, no real or mock data is being used, but this can be easily swapped in.

The buttons I'm using for accepting, declining, or canceling will more than likely benefit from being turned into a singular component.

## What is the change?
The only change made is to App. I added a redirect for all routes to the root '/', so that they are sent to '/dashboard'. This may be changed in the future.

## What does it fix?
No fixes.

## Where should the reviewer start?
App -> Dashboard -> TeeTimeContainer -> TeeTime -> InviteTypeSelect

## How should this be tested?
Via the browser as no Cypress tests yet exist.

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,9 +1,42 @@
...
-    <>
-      <h3>this is the dashboard</h3>
-    </>
+    <div className='dashboard'>
+      <span className='dashboard-title'>
+        <h1>My Tee Times</h1>
+      </span>
+      <span className='event-type-select'>
+        <button
+          className={teeTimeType === 'committed' ? 'event-type-btn' : 'event-type-btn unselected'}
```
Comment (mistercanderson) [inline src/components/Dashboard/Dashboard.js:16]: Nice use of state to determine the className here

File: src/components/TeeTime/TeeTime.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.hole-count {
+  grid-column: 3;
+  grid-row: 1;
+  text-align: right;
+}
+
+.host-name {
+  grid-column: 1 / span 2;
```
Comment (mistercanderson) [inline src/components/TeeTime/TeeTime.css:57]: You're like the grid master

File: src/components/TeeTime/TeeTime.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+.cancel {
+  background-color: rgb(212, 74, 68);
+  margin-left: 8px;
+  color: white;
+}
+
+.secondary-btn {
+  background-color: unset;
```
Comment (mistercanderson) [inline src/components/TeeTime/TeeTime.css:99]: I haven't used this unset value before, does it assign the opposite color to the button?

File: src/components/App/App.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import './App.css';
 import Header from '../Header/Header'
 import Dashboard from '../Dashboard/Dashboard'
 import React, { useState, useEffect } from 'react' 
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom'
 
 function App() {
   const [screenWidth, setScreenWidth] = useState(window.innerWidth);
```
Comment (mistercanderson) [inline src/components/App/App.js:8]: In order for this to work fully, I think we need to also add this: ```
const handleResize = () => setScreenWidth(window.innerWidth); 
useEffect(() => {
  window.addEventListener('resize', handleResize)
}, [])```

File: src/components/TeeTime/TeeTime.css (hunk 3 above)
Comment (jon-schlandt) [reply src/components/TeeTime/TeeTime.css:99]: Unset in this scenario is undoing the default background-color style for the button. Buttons default to having a background color, so unset removes this styling.

I use this a lot when I'm using media queries. Like for instance, if I have a div that on mobile view I want padding for, but on the desktop view I don't, I can use 'unset' in the media query to undo the padding I set on mobile view.

File: src/components/TeeTime/TeeTime.css [hunk 5]
```diff
@@ -67,21 +67,46 @@
...
+  align-items: center;
+  margin: 0 12px;
+  padding: 12px 0;
+  border-top: solid 1px rgba(0, 0, 0, .2);
 }
 
-.invitation-actions > button {
+.primary-btn {
+  width: fit-content;
+  height: fit-content;
```
Comment (errabun) [inline src/components/TeeTime/TeeTime.css:78]: Fit-content!  That is a new one for me, thanks for the new tool!

**Pull request merged**